src/build/render/features/tag/index.js

```diff
@@ -0,0 +1,133 @@
+const R = require("ramda");
+const {defAnchor, localizer, tagAnchor, html, detailsList, renderMarkdownInline} = require("../../components");
+const renderTagStructure = require("./tagStructure");
+
+const localizations = localizer({
+  parentTag: {
+    en: "Parent tag",
+    es: "Tag principal"
+  },
+  referencedBy: {
+    en: "Referenced by",
+    es: "Referenciado por",
+  },
+  childTags: {
+    en: "Child tags",
+    es: "Tags secundarias"
+  },
+  directReferences: {
+    en: "Direct references",
+    es: "Referencias directas"
+  },
+  inheritedReferences: {
+    en: (parent) => `${parent} references`,
+    es: (parent) => `Referencias de ${parent}`
+  }
+});
+
+function localizeThanks(ctx, thanks) {
+  return R.mapObjIndexed((forLangs, to) => {
+    const forMd = forLangs[ctx.lang];
+    return forMd ? [renderMarkdownInline(ctx, forMd)] : [];
+  }, thanks);
+}
+
+module.exports = async function(ctx) {
```

Comment:
All of this function used to be in one giant page render function, along with all the other render features. It's still long but at least it's a separated module now. I haven't really changed much about the rendering of tag structure and tag metabox content aside from localization support.